RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -24,7 +24,7 @@ DAClusterizerInZ_vect::DAClusterizerInZ_vect(const edm::ParameterSet& conf) {
   maxIterations_ = 1000;
   mintrkweight_ = 0.5;
 
-  // configurable debug outptut debug output
+  // configurable debug outptut
   verbose_ = conf.getUntrackedParameter<bool>("verbose", false);
```

Comment:
`verbose_` is only used at https://github.com/cms-sw/cmssw/blob/01849129ce26a43cd91423edd71d941843ab0a04/RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc#L50-L68

The same functionality could be replaced, in a more thread compliant way, by using the `DEBUG` / `DEBUG_LEVEL` structure which is used everywhere else in this code

Reply:
The very same apply to the other file DAClusterizerInZT_vect.cc